.editorconfig

```diff
@@ -0,0 +1,16 @@
+# top level config
+root = true
+
+# All files:
+# utf-8
+# Unix-style newlines
+# Remove end-of-file newlines
+# Soft indents (2 spaces)
+# Trim tailing whitespace
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = false
```

Comment:
Updated
On Wed, Aug 31, 2016 at 3:06 PM Blade Barringer <notifications@github.com>
wrote:

> In .editorconfig
> <https://github.com/HabitRPG/habitrpg/pull/7957#discussion_r77062615>:
>
> > @@ -0,0 +1,16 @@
> > +# top level config
> > +root = true
> > +
> > +# All files:
> > +# utf-8
> > +# Unix-style newlines
> > +# Remove end-of-file newlines
> > +# Soft indents (2 spaces)
> > +# Trim tailing whitespace
> > +[*]
> > +charset = utf-8
> > +end_of_line = lf
> > +insert_final_newline = false
>
> Actually, I reviewed the editorconfig I usually use and I go back on what
> I said. I prefer true as well 😄
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7957/files/23a50e0318fe1021eb59adf1ff60b0a41fbd4f76#r77062615>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_8xpPdBRA-AFeZjGuixRoAcuJms53ks5qld5ugaJpZM4Jx4Ke>
> .
>